src/components/Status/Status.js

```diff
@@ -1,67 +1,117 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
-import { Popover } from '@patternfly/react-core';
+import { Popover, Progress, ProgressVariant, ProgressSize } from '@patternfly/react-core';
 import { Icon, Button } from 'patternfly-react';
 import PropTypes from 'prop-types';
 
-export const Status = ({ icon, children }) => (
+import { STATUS_DANGER, STATUS_SUCCESS, STATUS_INFO } from '../../utils/status/common';
+
+const StatusField = ({ content }) => <div className="kubevirt-status__field">{content}</div>;
+StatusField.propTypes = {
+  content: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
+};
+
+export const StatusDescriptionField = ({ title, content }) => (
+  <StatusField
+    content={
+      <React.Fragment>
+        {title ? <div className="kubevirt-status__field-header">{title}</div> : ''}
+        <div className="kubevirt-status__field-description">{content}</div>
+      </React.Fragment>
+    }
+  />
+);
+StatusDescriptionField.defaultProps = {
+  title: null,
+};
+StatusDescriptionField.propTypes = {
+  title: PropTypes.string,
+  content: PropTypes.string.isRequired,
+};
+
+export const StatusLinkField = ({ title, linkTo }) => (
+  <StatusField
+    content={
+      <div className="kubevirt-status__field-link">
+        <Link to={linkTo} title={title}>
+          {title || linkTo}
+        </Link>
+      </div>
+    }
+  />
+);
+StatusLinkField.defaultProps = {
+  title: null,
+};
+StatusLinkField.propTypes = {
+  title: PropTypes.string,
+  linkTo: PropTypes.string.isRequired,
+};
+
+export const StatusProgressField = ({ title, progress, barType }) => {
+  const variants = {
```

Comment:
this object can be static

we could also export this or/and the constants from here.

 I am not sure the mapping is necessary though - we could have just used the original constants

Reply:
> I am not sure the mapping is necessary though - we could have just used the original constants

makes sense :smile: